tests/buildtest/inside_docker.rs

```diff
@@ -12,7 +12,15 @@ static CONTAINER_PREFIX: &str = "/outside";
 #[test]
 #[cfg(unix)]
 fn test_hello_world() {
+    // regression test for https://github.com/rust-lang/rustwide/issues/30
+    let already_present = Path::new("docker").exists();
+    if !already_present {
+        std::fs::create_dir("docker").unwrap();
+    }
     execute("buildtest::test_hello_world").unwrap();
+    if !already_present {
+        std::fs::remove_dir("docker").unwrap();
+    }
```

Comment:
This will leave a stray `docker` directory if `execute()` fails. This test is also `#[cfg(unix)]`, so we wouldn't execute it on Windows while this is a bug that could happen there.

I'd create a new standalone test (`test/issue_30.rs`) that creates a temporary directory, `std::env::set_current_dir`s into it, creates a `true` directory and tries to call the global `true` binary. This needs to be in a standalone test to avoid the `std::env::set_current_dir` affecting other tests.